example_conf.yml

```diff
@@ -0,0 +1,47 @@
+s:
+  - type: Speed
+    style: hbar
+    x: 30
+    y: 985
+    units: 'imperial'
```

Comment:
Agreed but there is currently no way to propagate global settings to appropriate widgets. I left it this way for now because it's only relevant for the single map widget we have.